test/AuthTokenVerifier.test.ts

```diff
@@ -45,56 +43,53 @@ describe("Testing AuthTokenVerifier", () => {
       sub: secondDID
     }
 
-    firstSigner = new JWT(firstKeys);
-    secondSigner = new JWT(secondKeys)
-    firstClaimToken = await firstSigner.sign(firstPayload, { algorithm: 'ES256' });
-    secondClaimToken = await secondSigner.sign(secondPayload, { algorithm: 'ES256' });
-  })
-
-  it("Checks if basic signing verification works", async () => {
-    const testSignature = await secondSigner.sign('Initial signed content', { algorithm: 'ES256' })
-    const secondPubKey = secondKeys.publicKey
-    const decoded = await secondSigner.verify(testSignature, secondPubKey)
-    
-    assert.strictEqual(decoded, 'Initial signed content')
+    firstClaim = await createSignWithEthersSigner(firstWallet)(firstPayload);
+    secondClaim = await createSignWithEthersSigner(secondWallet)(secondPayload);
   })
 
   it("should authenticate first DID with first Claim issuer", async () => {
     
-    tokenAuthenticationVerifier = new AuthTokenVerifier(firstKeys.privateKey, firstDocument);
-    const did = await tokenAuthenticationVerifier.verify(firstClaimToken, firstPayload.iss)
+    verifier = new AuthTokenVerifier(firstDocument);
+    const did = await verifier.verify(firstClaim)
 
-    assert.strictEqual(did, issuerDID)
+    assert.strictEqual(did, firstDID)
   })
 
   it("should reject the second DID authentication with first Claim issuer", async () => {
```

Comment:
@jrhender is that description fits to what you have in mind ?
```suggestion
  it("should reject authentication with mismatching DID doc ", async () => {
```

Reply:
Yes, exactly. I think these kind of test descriptions are much better

Reply:
I agree